blueprints/ember-cli-typescript/index.js

```diff
@@ -32,7 +32,10 @@ module.exports = {
     let inRepoAddons = (this.project.pkg['ember-addon'] || {}).paths || [];
     let hasMirage = 'ember-cli-mirage' in (this.project.pkg.devDependencies || {});
     let isAddon = this.project.isEmberCLIAddon();
-    let includes = ['app', isAddon && 'addon', 'tests', 'types'].concat(inRepoAddons).filter(Boolean);
+    let isMU = this._detectMU();
```

Comment:
does javascript support unicode for variables? `this._isμ`? :trollface: 

I actually just looked up upper case mu/μ, and it's pretty boring. (capital M). lame.